src/auth-service/utils/create-user.js

```diff
@@ -1064,6 +1107,82 @@ const join = {
       };
     }
   },
+
+  deleteMobileUserData: async (request) => {
+    try {
+      const { body } = request;
+      let { uid } = body;
+      let { ct } = body;
```

Comment:
could we use more descriptive variable names? `ct` can easily be confusing for future maintainers? This applies to all occurances of `ct` in the code. And similarly name variable names. @BenjaminSsempala

Reply:
Thanks I've updated this. @Baalmart